docs/2.8.0/docs/query/pqs-user-guide.rst

```diff
@@ -623,7 +628,7 @@ information.
 A Daml transaction is a collection of events that take effect on the
 ledger atomically. However it needs to be noted that for performance
 reasons these transactions are written to the datastore *in parallel*,
-and although the datastore is written to in a purely append-only fashion,
+and while the datastore is written to in a purely append-only fashion,
```

Comment:
Revert -- when indicating contrast, "although" is preferred over "while" (https://developers.google.com/style/word-list#while)

(In https://github.com/digital-asset/docs.daml.com/pull/467/ I changed "whilst/while" to "although")